src/Symfony/Component/Finder/Tests/Iterator/FilePathsIteratorTest.php

```diff
@@ -44,7 +44,7 @@ public function getSubPathData()
                     $tmpDir.DIRECTORY_SEPARATOR.'test.php' => $tmpDir.DIRECTORY_SEPARATOR.'test.php',
                     $tmpDir.DIRECTORY_SEPARATOR.'toto' => $tmpDir.DIRECTORY_SEPARATOR.'toto',
                 ),
-                array( // subPaths
+                array(// subPaths
```

Comment:
I kind of like the space here. Another fix could be to put the comment on another line:

```php
array(
    // subPaths
    ...
```